### PR TITLE
Add Update to facilitate CA renewals.

### PR DESCRIPTION
**Pull request by kisom**

### CODE REVIEW

File: initca/initca.go [hunk 1]
```diff
@@ -220,3 +219,17 @@ var CAPolicy = func() *config.Signing {
 		},
 	}
 }
+
+// CopyAndResign copies the CA certificate, updates the NotBefore and
+// NotAfter fields, and then re-signs the certificate.
+func CopyAndResign(ca *x509.Certificate, priv crypto.Signer) (cert []byte, err error) {
```
Comment (lziest) [inline initca/initca.go]: Re-issue? Resign is kinda... negative

Comment (lziest) [reply initca/initca.go]: Or just renew?

Comment (kisom) [reply initca/initca.go]: We're also not issuing anything, really. Renamed to update. Now we wait for the tests to re-run.

File: initca/initca.go [hunk 2]
```diff
@@ -220,3 +219,17 @@ var CAPolicy = func() *config.Signing {
 		},
 	}
 }
+
+// Update copies the CA certificate, updates the NotBefore and
+// NotAfter fields, and then re-signs the certificate.
+func Update(ca *x509.Certificate, priv crypto.Signer) (cert []byte, err error) {
```
Comment (lziest) [inline initca/initca.go]: shall we check the CA bit of the input cert? And basically we don't need `Renew...` function any more, right?

Comment (kisom) [reply initca/initca.go]: I don't think it really matters. If the CA bit isn't set, you're just ending up with a self-signed cert anyways. This is really just a simple copy-and-update function. The Renew function is used elsewhere right now, so it should be deprecated, not removed right away.

**Pull request merged**